innkeeper/client.go

```diff
@@ -374,14 +409,31 @@ func (c *Client) requestData(authRetry bool, url string) ([]*routeData, error) {
 	return result, err
 }
 
+func (c *Client) LoadAndParseAll() ([]*eskip.RouteInfo, error) {
+	routeList, err := c.LoadAll()
+	if err != nil {
+		return nil, err
+	}
+
+	routeInfos := []*eskip.RouteInfo{}
+
+	for _, route := range routeList {
+		routeInfo := &eskip.RouteInfo{
+			*route, nil}
```

Comment:
see my comments for the `RouteInfo` type, in the eskip package per this pull request. I think this `LoadAndParseAll` makes only sense in case of the etcd client. (the parser error is `nil` in this line.)